action_transaction_test.go

```diff
@@ -187,6 +187,37 @@ func Test_RevertTransaction(t *testing.T) {
 	})
 }
 
+func Test_RecordTransaction(t *testing.T) {
+	//var secondTransaction *Transaction
+	t.Run("record transaction", func(t *testing.T) {
+		ctx, client, _ := initSimpleTestCase(t)
+		// given
+		draftTransaction := newDraftTransaction(
+			testXPub, &TransactionConfig{
+				Outputs: []*TransactionOutput{{
+					To:       "1A1PjKqjWMNBzTVdcBru27EV1PHcXWc63W",
+					Satoshis: 1000,
+				}},
+				ChangeNumberOfDestinations: 1,
+				Sync: &SyncConfig{
+					Broadcast:        true,
+					BroadcastInstant: false,
+					PaymailP2P:       false,
+					SyncOnChain:      false,
+				},
+			},
+			append(client.DefaultModelOptions(), New())...,
```

Comment:
I would like to ask you to implement tests for the remaining cases for validation purposes, as long as they are easy to write. If not please ignore this comment

Reply:
Added tests